src/assets/scripts/aircraft/AircraftFlightManagementSystem.js

```diff
@@ -56,13 +64,21 @@ const WAYPOINT_WITHIN_LEG = 1;
 export default class AircraftFlightManagementSystem {
     constructor(options) {
         this.my_aircrafts_eid = options.aircraft.eid;
+
+        // TODO: we should remove this reference and instead supply methods that the aircraft can call via the fms
         this.my_aircraft = options.aircraft;
+
         this.legs = [];
+
         this.current = [0, 0]; // [current_Leg, current_Waypoint_within_that_Leg]
```

Comment:
usually, not cool.  But there should be doc blocks for each property which is why, I think, I left the spacing here.

I'll add those now.